tests/testthat/test-verb-arrange.R

```diff
@@ -122,6 +123,7 @@ test_that("arranges captures DESC", {
 })
 
 test_that("multiple arranges combine", {
+  skip("not clear whether test makes sense")
```

Comment:
`out` is defined twice and I don't think that multiple `arrange()` should really combine.